stub-oidc-op.yml

```diff
@@ -1,10 +1,10 @@
 server:
   applicationConnectors:
     - type: http
-      port: 5510
+      port: ${APPLICATION_PORT:-5510}
   adminConnectors:
     - type: http
       port: 5511
 
 
-redisURI: ${REDIS_URI:-192.168.204.170:6379/0}
+redisURI: ${REDIS_URI:-"redis://192.168.204.170:6379"}
```

Comment:
is this the IP address of the redis instance on PaaS?

Reply:
Ahh no, that's the default when I run it locally. It gets the Redis URI from the RedisService.java by parsing the VCAP_SERVICES env variable which is PAAS thing - https://docs.cloud.service.gov.uk/deploying_services/redis/#connect-to-a-redis-service-from-your-app